lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Operations/LMFeedEditPostOperation.swift

```diff
@@ -97,31 +97,31 @@ final class LMFeedEditPostOperation {
                 .description(linkAttachment.description ?? "")
                 .url(linkAttachment.url)
             
-            let attachmentMeta = AttachmentMeta()
-                .ogTags(ogTags)
+            var attachmentMeta = AttachmentMeta.Builder()
+            attachmentMeta = attachmentMeta.ogTags(ogTags)
             
             let attachment = Attachment()
                 .attachmentType(.link)
-                .attachmentMeta(attachmentMeta)
+                .attachmentMeta(attachmentMeta.build())
             
             attachments.append(attachment)
         }
         
         if let poll {
-            let attachmentMeta = AttachmentMeta()
-                .entityID(poll.id)
-                .title(poll.question)
-                .expiryTime(poll.expiryTime)
-                .pollOptions(poll.options.map({ $0.option }))
-                .multiSelectState(poll.pollSelectType.apiKey)
-                .pollType(poll.isInstantPoll ? "instant" : "deferred")
-                .multSelectNo(poll.pollSelectCount)
-                .isAnonymous(poll.isAnonymous)
-                .allowAddOptions(poll.allowAddOptions)
+            var attachmentMeta = AttachmentMeta.Builder()
+            attachmentMeta = attachmentMeta.entityID(poll.id)
+            attachmentMeta = attachmentMeta.title(poll.question)
+            attachmentMeta = attachmentMeta.expiryTime(poll.expiryTime)
+            attachmentMeta = attachmentMeta.pollOptions(poll.options.map({ $0.option }))
+            attachmentMeta = attachmentMeta.multiSelectState(poll.pollSelectType.apiKey)
+            attachmentMeta = attachmentMeta.pollType(poll.isInstantPoll ? "instant" : "deferred")
+            attachmentMeta = attachmentMeta.multSelectNo(poll.pollSelectCount)
+            attachmentMeta = attachmentMeta.isAnonymous(poll.isAnonymous)
+            attachmentMeta = attachmentMeta.allowAddOptions(poll.allowAddOptions)
```

Comment:
why we have to do this way?

Reply:
AttachmentMeta was not on Builder pattern, have to shift it to builder pattern.